app/code/Magento/QuoteGraphQl/Model/Resolver/UpdateCartItems.php

```diff
@@ -77,6 +77,7 @@ public function resolve(Field $field, $context, ResolveInfo $info, array $value
         try {
             $this->updateCartItems->processCartItems($cart, $cartItems);
             $this->cartRepository->save($cart);
+            $cart = $this->getCartForUser->execute($maskedCartId, $context->getUserId(), $storeId);
```

Comment:
Could we try to achieve the same result without an additional database query? I mean, we already have all the details about the shopping cart. Most likely, we don't need an additional DB fetch, but the recalculation process that happens upon the DB fetch.